### PR TITLE
Various tested games

### PR DESCRIPTION
**Pull request by daddeltrotter**
Here I go again :)

### CODE REVIEW

File: tweaks/steam.yaml [hunk 1]
```diff
@@ -1107,8 +1165,8 @@
 "207490": # Rayman Origins
   compat_tool: proton_63
 
-"1158450": # GALAXIUM
-  compat_tool: proton_63
+"1067540": # RÖki
```
Comment (alkazar) [inline tweaks/steam.yaml:1168]: This is actually a duplicate.

Comment (daddeltrotter) [reply tweaks/steam.yaml:1168]: GALAXIUM is wrongly sorted. I move it up to line 594.

Comment (alkazar) [reply tweaks/steam.yaml:1168]: I mean Roki, there is another instance of it later in the file.

**Comment by alkazar** [comment]
Amazing work as always, thank you!

**Pull request merged**